From: monsta <monsta@inbox.ru>
Date: Mon, 21 Nov 2016 23:55:03 +0300
Subject: add missing #include

---
 src/gpm-networkmanager.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/gpm-networkmanager.h b/src/gpm-networkmanager.h
index b60208b..e7eaa4c 100644
--- a/src/gpm-networkmanager.h
+++ b/src/gpm-networkmanager.h
@@ -22,6 +22,8 @@
 #ifndef __GPM_NETWORKMANAGER_H
 #define __GPM_NETWORKMANAGER_H
 
+#include <glib.h>
+
 G_BEGIN_DECLS
 
 gboolean	gpm_networkmanager_sleep		(void);
